Kickstarter-iOS/Views/Cells/RewardCell.swift

```diff
@@ -105,14 +110,26 @@ final class RewardCell: UICollectionViewCell, ValueCell {
         transform = .identity
       }
 
-      _ = self
+      _ = self.rewardCardContainerView
```

Comment:
What's this for? 👀

Reply:
Previously we were transforming the whole cell which was causing a visual gap between the card and this divider line. Now we just transform the `rewardCardContainerView` to fix that.

Reply:
Nice! Thanks for fixing 🙏